tests/UI/specs/UIIntegration_spec.js

```diff
@@ -498,6 +498,12 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
         }, done);
     });
 
+    it('should load the config file page correctly', function (done) {
+        expect.screenshot('admin_diagnostics_configfile').to.be.captureSelector('.pageWrap', function (page) {
+            page.load("?" + generalParams + "&module=Diagnostics&action=configfile");
+        }, done);
+    });
+
```

Comment:
Thought about the toggled ones too but then I found I don't want to overdo it as it is not really critical etc